components/raftstore/src/store/worker/raftlog_gc.rs

```diff
@@ -115,9 +96,12 @@ impl<EK: KvEngine, ER: RaftEngine> Runner<EK, ER> {
             panic!("failed to sync kv_engine in raft_log_gc: {:?}", e);
         });
         RAFT_LOG_GC_KV_SYNC_DURATION_HISTOGRAM.observe(start.saturating_elapsed_secs());
+
+        fail::fail_point!("worker_gc_raft_log");
```

Comment:
I don't think it's the same as before if configured with "return". In the past, only `batch_gc` is skipped. But now all the following processing are skipped.